FullCircle/FullCircle/FirestoreService.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  FirestoreService.swift
+//  FullCircle
+//
+//  Created by David Rifkin on 8/4/20.
+//  Copyright © 2020 FullCircle. All rights reserved.
+//
+
+import Foundation
+import FirebaseFirestore
+
+class FirestoreService {
+
+    //MARK: Service Setup
+    static let manager = FirestoreService()
+        
+    private let db = Firestore.firestore()
+    private init () {}
+
+    //MARK: Model Types
+    enum ModelTypes: String {
+        case organization
+        case action
+    }
+    
+    //MARK: Organizations
+    func getAllOrganizations(completion: @escaping (Result<[Organization], Error>) -> ()) {
+        db.collection(ModelTypes.organization.rawValue).getDocuments { (snapshot, error) in
+            if let error = error {
+                completion(.failure(error))
+            } else {
+                let organizations = snapshot?.documents.compactMap({ (snapshot) -> Organization? in
+                    let orgID = snapshot.documentID
+                    let org = Organization(from: snapshot.data(), id: orgID)
+                    return org
+                })
+                completion(.success(organizations ?? []))
+            }
+        }
+    }
+    
+    //MARK: Actions. TODO: Add getAll(forOrganization:) and getAll(forUser:)
+    func getAllActions(completion: @escaping (Result<[Action], Error>) -> ()) {
+        db.collection(ModelTypes.action.rawValue).getDocuments { (snapshot, error) in
+            if let error = error {
+                completion(.failure(error))
+            } else {
+                let organizations = snapshot?.documents.compactMap({ (snapshot) -> Action? in
```

Comment:
Hi @davidlawrencer !

1. Wanted to confirm that no actions should be loaded at the moment is that right? I see we don't have a collections of all actions in FB (talk more about how we want to tackle that on Thurs)

2. Super minor copy/paste typo but I think you mean "actions" instead of "organizations" here right?

Reply:
Yes to both! @msystang for #1 we'll just have to set up a bunch of dummy data at some point.

Reply:
@msystang just pushed changes for #2